Harvest.Web/ClientApp/src/Quotes/WorkItemsForm.tsx

```diff
@@ -38,22 +38,30 @@ export const WorkItemsForm = (props: Props) => {
       // new rate selected, update the work item with defaults
       props.updateWorkItems({
         ...workItem,
-        description: rate.description,
+        description: requiresCustomDescription(rate.unit)
+          ? ""
+          : rate.description,
         rateId,
         rate: rate.price,
+        unit: rate.unit,
         total: 0,
       });
     }
   };
 
+  // TODO: Determine a better way of working out which other options need extra description text
+  const requiresCustomDescription = (unit: string) => {
+    return props.category === "other" && unit === "Unit";
```

Comment:
@srkirkland Other not other ?

Reply:
@jSylvestre has to be other lowercase to work.  Though it's just an optimization we don't even need that part of the check. but i'll leave for now until we determine a better way to detect these cases.